.github/PULL_REQUEST_TEMPLATE/pull-request-template.md

```diff
@@ -37,13 +37,16 @@ If the issue is not fully described in Github, add more information here (justif
 Provide a clear description of how this change was tested.
 At minimum this should include proof that a computer has executed the changed lines.
 Ideally this should include an automated test or an explanation as to why this change has no tests.
+To execute all automated regressions tests start the script /WinCC_OA_Test/executeTests.cmd
```

Comment:
> We don't want OS-specific code here. Why only Windows?

because I have only windows PC and can not test shell script. When we want it, it shall be greate, when we integrate it in the docker image to test the changes automatically in each build. And exact that the goal.